Tests/SentryTests/SwiftUI/SentryRedactModifierTests.swift

```diff
@@ -0,0 +1,18 @@
+#if canImport(UIKit) && canImport(SwiftUI)
+@testable import Sentry
+@testable import SentrySwiftUI
+import SwiftUI
+import XCTest
+
+class SentryRedactModifierTests: XCTestCase {
+
+    func testViewRedate() throws {
```

Comment:
`l`: This looks like a typo, or am I not getting it?

Reply:
Typo. Thanks!

Reply:
```suggestion
    func testViewRedacted() throws {
```